server/app/services/applicant/ApplicantService.java

```diff
@@ -1304,9 +1310,12 @@ private ApplicationPrograms relevantProgramsForApplicantInternal(
                 getApplicantMayBeEligibleStatus(applicant, program));
           }
 
-          if (program.isCommonIntakeForm()) {
+          ProgramType programType = program.programType();
+          if (programType.equals(ProgramType.COMMON_INTAKE_FORM)) {
             relevantPrograms.setCommonIntakeForm(applicantProgramDataBuilder.build());
-          } else {
+          } else if (programType.equals(ProgramType.DEFAULT)
```

Comment:
This is the functionality change. The rest is just passing `request` to the caller